### DISCUSSION TITLE
How to Import Employee data

### DISCUSSION THREAD

**Discussion by MrcellSbst** [horilla-opensource/horilla#327] [Q&A]
i want to import employee data, but it keep showing error in importerror file:

Title Error
The provided titles don't match the default titles.

help me pls

**Comment by horilla-opensource** [comment]
> i want to import employee data, but it keep showing error in importerror file:
> 
> Title Error The provided titles don't match the default titles.
> 
> help me pls

Hi @MrcellSbst ,
Can you confirm whether you are using the same template which was downloaded on clicking _import_ for importing the record and no changes have been made into the headers of the excel sheet?

**Reply by MrcellSbst** [reply]
in reporting manager, what should i filled it with? badge number? name?

**Reply by horilla-opensource** [reply]
Hi @MrcellSbst ,
You need to add the reporting manager name in the column.

**Comment by MrcellSbst** [comment]
yes, i'm using the template from the server and nothing changed on the
header
